recipes/leftbar/general_row/general_row_constants.js

```diff
@@ -0,0 +1,32 @@
+export const LEFTBAR_GENERAL_ROW_TYPES = {
+  INBOX: 'inbox',
+  CONTACTS: 'contacts',
+  CHANNELS: 'channels',
+  THREADS: 'threads',
+  LOCKED_CHANNEL: 'locked channel',
+  CONTACT_CENTER: 'contact center',
+  DIALBOT: 'dialbot',
```

Comment:
erk

<img width="524" alt="image" src="https://user-images.githubusercontent.com/1165933/220527102-16acffa4-3dd0-4619-9ed9-e35d7f7d8884.png">

Reply:
gross

Reply:
should be fixed.